packages/remark-wiki-link/test/remarkWikiLink.spec.ts

```diff
@@ -361,9 +413,131 @@ describe("remark-wiki-link", () => {
     });
   });
 
+  describe("Links with special characters", () => {
```

Comment:
I don't think we need a separate test for each of these cases. I'd merge them into one with some crazy wiki link having accented letters, spaces, dashes and a bunch of other signs.